util/reggen/register.py

```diff
@@ -543,3 +608,53 @@ def _asdict(self) -> Dict[str, object]:
             rd['storage_err_alert'] = self.storage_err_alert
 
         return rd
+
+    def apply_alias(self, alias_reg: object, where: str) -> None:
+        '''
+        Compare all attributes and replace overridable values.
+
+        This updates the overridable register and field attributes with the
+        alias values and ensures that all non-overridable attributes have
+        identical values.
+        '''
+        if not isinstance(alias_reg, Register):
+            raise ValueError('alias_reg is not of type Register')
+
+        for key, val in self.__dict__.items():
```

Comment:
As above, I think this would probably be clearer if we just expanded out the fields.